Makefile

```diff
@@ -818,3 +833,22 @@ release-notes: /dev/null
 .PHONY: checksums
 checksums:
 	sha256sum ./dist/argo-*.gz | awk -F './dist/' '{print $$1 $$2}' > ./dist/argo-workflows-cli-checksums.txt
+
+# dev container
+
+$(TOOL_DEVCONTAINER): Makefile
+	npm list -g @devcontainers/cli@0.75.0 > /dev/null || npm i -g @devcontainers/cli@0.75.0
+
+.PHONY: devcontainer-build
+devcontainer-build: $(TOOL_DEVCONTAINER)
+	devcontainer build \
+		--workspace-folder . \
+		--config .devcontainer/builder/devcontainer.json \
+		--platform $(TARGET_PLATFORM) \
```

Comment:
I think we need set buildx context then build it to allow it to be multiarch

Reply:
I think you mean we need to use the [docker container build driver](https://docs.docker.com/build/builders/drivers/docker-container/). That's handled by the `docker/setup-buildx-action` action, and you can see that running in https://github.com/argoproj/argo-workflows/actions/runs/14185152964/job/39739033877?pr=14341

Specifically, the action executes this command to create the driver and use it as the default:
```
  /usr/bin/docker buildx create --name builder-fe8bbb6f-d828-4446-a45a-4e7df05ff27d --driver docker-container --buildkitd-flags --allow-insecure-entitlement security.insecure --allow-insecure-entitlement network.host --use
``